pom.xml

```diff
@@ -52,53 +52,49 @@
             <scope>system</scope>
             <systemPath>${basedir}/lib/icmp4j-1018.jar</systemPath>
         </dependency>
-        <dependency>
-            <groupId>org.apache.sshd</groupId>
-            <artifactId>sshd-core</artifactId>
-            <version>2.2.0</version>
-        </dependency>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-            <scope>test</scope>
-        </dependency>
 
         <!--dependencies of symphony-dal-commons-->
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
             <version>4.2.4.RELEASE</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.9.10.3</version>
+            <version>2.10.0.pr1</version>
```

Comment:
This is preview version, we should not use preview versions, only released. I'd recommend to use the same versions as Symphony uses in corresponding release.

Reply:
Agree, versions will be aligned with target version 4.9.0